zipline/data/data_portal.py

```diff
@@ -1239,6 +1239,29 @@ def get_simple_transform(self, asset, transform_name, dt, data_frequency,
 
             return ret
 
+    def get_current_future_chain(self, continuous_future, dt):
+        """
+        Retrieves the future chain for the contract at the given `dt` according
+        the `continuous_future` specification.
+
+        Returns:
+        future_chain : [Future]
```

Comment:
Had meant to follow the formatting of methods like `BcolzDailyBarWriter.write_csv`, e.g. `asset_map : dict[int -> str]`, had left off the return type though.

Will change to to `list[Future]`.